docs/reST/ref/image.rst

```diff
@@ -167,12 +167,26 @@ following formats.
    | :sg:`frombuffer(bytes, size, format) -> Surface`
 
    Create a new Surface that shares pixel data directly from a bytes buffer.
-   This method takes the same arguments as ``pygame.image.fromstring()``, but
+   This method takes similar arguments to ``pygame.image.fromstring()``, but
    is unable to vertically flip the source data.
 
    This will run much faster than :func:`pygame.image.fromstring`, since no
    pixel data must be allocated and copied.
 
+   It accepts the following 'format' arguments:
```

Comment:
If this is the case can you add `_format = Literal['p', 'RGB', 'BRG', 'RGBX', RGBA', 'ARGB']` to the image.pyi and then replace `format: str` with `format: _format`. You can see the example of it in pixelcopy.pyi

Reply:
I have now done this for all of the to/from functions to match their code.